QUARKUS-3004.md

```diff
@@ -0,0 +1,33 @@
+# QUARKUS-3004: ARM support for JVM mode on OCP
+
+Jira: https://issues.redhat.com/browse/QUARKUS-3004
+
+The goal of this feature is to provide production support for Red Hat build of Quarkus running in JVM mode on OpenShift
+on most commonly used server side 64-bit ARM architectures.
+
+## Scope of the testing
+To verify that RHBQ 3 properly functions on ARM achitectures in JVM mode, the OpenShift test suite will have to pass 
+with OpenJDK 17.
+
+### Impact on test suites and testing automation
+* OCP testings
+  * OCP4 IPI install/destroy on AWS on ARM job will have to be implemented
+  * A pipeline will have to be implemented in extended platforms that will setup OCP on ARM, run Quarkus QE OpenShift 
+    testsuite on it and then clean it up
+  * Determine which of operators and testing services are supported on OCP on ARM
+  * Open questions
```

Comment:
As OCP is certified on aarch64 I would assume S2I works there.\

https://access.redhat.com/articles/4912821  - Cloud Providers / Infrastructure says:
```
Amazon Web Services (AWS) | TESTED
Red Hat OpenShift Service on AWS (ROSA) | TESTED
```